cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/queue/SimpleQueueSpecificationGeneratorTest.java

```diff
@@ -81,7 +82,7 @@ public void testQueueSpecificationGenWithToyApp() throws Exception {
     ApplicationSpecification newSpec = adapter.fromJson(adapter.toJson(appSpec));
 
     QueueSpecificationGenerator generator =
-      new SimpleQueueSpecificationGenerator(Id.Application.from(TEST_NAMESPACE_ID, newSpec.getName()));
+      new SimpleQueueSpecificationGenerator(new ApplicationId(TEST_NAMESPACE_ID, newSpec.getName()));
```

Comment:
can you remove the constant ``TEST_NAMESPACE_ID`` and instead use ``NamespaceId.DEFAULT.app(newSpec.getName())``